decksite/api.py

```diff
@@ -18,6 +18,20 @@ def deck_api(deck_id):
     blob = deck.load_deck(deck_id)
     return return_json(blob)
 
+@APP.route('/api/randomlegaldeck/')
+def random_deck_api():
+    blob = deck.random_legal_deck()
+    return return_json(blob)
+
+@APP.route('/api/randomlegaldeckurl/')
```

Comment:
This seems unneccessary

Reply:
Just use the above endpoint and extract the url for it

Reply:
Removed /api/randomlegaldeckurl and added url attribute to the returned data from /api/randomlegaldeck